app/views/posts/index.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="post-section">
+  <% content_for :title, "#{@user.name}'s Posts" %>
+  <%= render "users/user", user: @user %>
+  <% @user.posts.each do | post | %>
+    <%= link_to user_post_path(@user, post) do %>
+      <%= render "post", post: post %>
+
+      <div>
+        <h4>
+          <span>Comments</span>
+          <span>for Post: </span>
+          <span><%="#{post.title}"%> </span>
+        </h4>
+        <div class='postComments'>
+          <% post.recent_comments.each do |comment| %>
+            <div>
+              <p>
+                <span><strong><%= comment.user.name%>:</strong></span>
+                <span><%= comment.text %></span>
```

Comment:
- [ ]  You have done a great job, but there still a room of improvement. I could run the seeds your created so that I could test your app,  but couldn't create both posts and comments. I could see list of users and his details but did see recent 3 posts and 5 comments appearing. Fix this issue and make sure that you following the provided wireframe structure.

Reply:
Thank you for your beneficial suggestion, as you can see in the following video all the mentioned functionalities were added to the project, but I created all the posts and comments manually and now I'll remove the seed file, to fix the issue.